openmc/plotter.py

```diff
@@ -0,0 +1,537 @@
+                    grid = np.union1d(grid, elastic.x)
+                    if elastic.x[-1] > sab_Emax:
+                        sab_Emax = elastic.x[-1]
+                sab_funcs.append(elastic)
+            if sab.inelastic_xs:
+                inelastic = sab.inelastic_xs[sabT]
+                grid = np.union1d(grid, inelastic.x)
+                if inelastic.x[-1] > sab_Emax:
+                        sab_Emax = inelastic.x[-1]
+                sab_funcs.append(inelastic)
+            energy_grid = grid
+        else:
+            energy_grid = nuc.energy[nucT]
+
+        for i, mt_set in enumerate(mts):
+            # Get the reaction xs data from the nuclide
+            funcs = []
+            op = ops[i]
+            for mt in mt_set:
+                if mt == 2:
+                    if sab_name:
+                        # Then we need to do a piece-wise function of
+                        # The S(a,b) and non-thermal data
+                        sab_sum = openmc.data.Sum(sab_funcs)
+                        pw_funcs = openmc.data.Regions1D(
+                            [sab_sum, nuc[mt].xs[nucT]],
+                            [sab_Emax])
+                        funcs.append(pw_funcs)
+                    else:
+                        funcs.append(nuc[mt].xs[nucT])
+                elif mt == 5 and mt in nuc:
+                    # Only consider the (n,misc) products if neutrons are
+                    # included in the outgoing channel since (n,misc) is only
+                    # explicitly needed for scatter cross sections.
+                    for prod in nuc[mt].products:
+                        if prod.particle == 'neutron' and \
+                            prod.emission_mode in ('total', 'prompt'):
+                            if yields[i]:
+                                func = openmc.data.Combination(
+                                        [nuc[mt].xs[nucT], prod.yield_],
+                                        [np.multiply])
+                            else:
+                                func = nuc[mt].xs[nucT]
+
+                            funcs.append(func)
+                            break
+                    else:
+                        funcs.append(lambda x: 0.)
+
+                elif mt in nuc:
+                    if yields[i]:
+                        for prod in nuc[mt].products:
+                            if prod.particle == 'neutron' and \
+                                prod.emission_mode in ('total', 'prompt'):
+                                func = openmc.data.Combination(
+                                    [nuc[mt].xs[nucT], prod.yield_],
+                                    [np.multiply])
+                                funcs.append(func)
+                                break
+                        else:
+                            # Assume the yield is 1
+                            funcs.append(nuc[mt].xs[nucT])
```

Comment:
Yeah, in ENDV/B-VII.0 from MCNP, NNDC ENDF/B-VII.1 and JEFF 3.2, MT=18 of U-234 uses this line.  MT=18 is a summed product on the python side, but at this moment there was no product pulled from total_nu.  I added that in and modified this code to read it, and then reorganized this block so we first look for a total product (which now works for all fissions) and failing that, sum the yields from delayed and prompt products.